src/Scene/Node.js

```diff
@@ -24,6 +24,7 @@ define('Scene/Node', [], function() {
         this.screenSpaceError = 0.0;
         this.loaded = false;
         this.pendingSubdivision = false;
+        this.pendingLayers = {};
```

Comment:
Given that `getDownScaledLayer` will always return the same id *until* it has been updated, maybe we can use a simple `bool` here?
(just a suggestion, doens't matter much)